src/components/ResearchGroupCard/style.scss

```diff
@@ -1,23 +1,120 @@
-.App {
-  text-align: center;
+$card-header-font: 24px;
+$card-header-height: 64px;
+
+.card{
+    text-align: center;
+    font-family: 'Futura', sans-serif;
+    background-color: #fafafa;
+    border-style: solid;
+    border-color: #cecece;
+    margin: 10px;
+    height: 450px;
+    min-width: 200px;
+    overflow-wrap: break-word;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+
+.card_professor{
+    text-align: left;
+    color: #5dadf7;
+    margin-left: 20px;
+    margin-right: 20px;
+    font-size: 16px;
+    height: 5%;
+    margin-top: 20px;
+    overflow-y: hidden;
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 40vmin;
+.card_research-group{
+    text-align: left;
+    font-weight: bold;
+    margin-left: 20px;
+    margin-right: 20px;
+    margin-top: 2px;
+    margin-bottom: 15px;
```

Comment:
Replace this and other margins with margin shorthand:

`margin: <top> <right> <bottom> <left>` so:
`margin: 2px 20px 15px 20px;`